help_to_heat/frontdoor/views.py

```diff
@@ -805,7 +812,102 @@ def get_context(self, request, session_id, *args, **kwargs):
         eligible_schemes = eligibility.calculate_eligibility(session_data)
         _ = interface.api.session.save_answer(session_id, "schemes", {"schemes": eligible_schemes})
         eligible_schemes = tuple(schemas.schemes_map[scheme] for scheme in eligible_schemes if not scheme == "ECO4")
-        return {"eligible_schemes": eligible_schemes}
+        supplier_name = SupplierConverter(session_id).get_supplier_on_general_pages()
+
+        is_in_park_home = session_data.get("park_home", "No") == "Yes"
+        is_solid_walls = session_data.get("wall_type") in [
+            "Solid walls",
+            "Mix of solid and cavity walls",
+            "I do not know",
+        ]
+        is_cavity_walls = session_data.get("wall_type") in [
+            "Cavity walls",
+            "Mix of solid and cavity walls",
+            "I do not know",
+        ]
+        is_not_on_benefits = session_data.get("benefits") == "No"
+        is_wall_insulation_present = not session_data.get("wall_insulation") in [
+            "Some are insulated, some are not",
+            "No they are not insulated",
+            "I do not know",
+        ]
+        is_income_above_threshold = session_data.get("household_income") == "£31,000 or more a year"
+        is_social_housing = session_data.get("own_property") == "No, I am a social housing tenant"
+        is_loft_present = session_data.get("loft") == "Yes, I have a loft that has not been converted into a room"
+        is_there_access_to_loft = session_data.get("loft_access") == "Yes, there is access to my loft"
+        is_loft_insulation_over_threshold = session_data["loft_insulation"] in [
+            "I have more than 100mm of loft insulation",
+            "I do not know",
+        ]
+        is_loft_insulation_under_threshold = session_data["loft_insulation"] in [
+            "I have up to 100mm of loft insulation",
+            "I do not know",
+        ]
+
+        text_flags = {
+            "park_home_text": is_in_park_home and not is_social_housing,
+        }
+        text_flags.update(
+            {
+                "cavity_wall_text": text_flags.get("park_home_text") is False
```

Comment:
not sure why some of these use `is` and some are used as boolean values? don't think `is` is needed?

also if these are all boolean flags could their name reflect that? ie `show_cavity_wall_text` vs `cavity_wall_text` (sounds like it'd be a text value)